### PR TITLE
[Salt-PR 2: Electric Boogaloo] Yet another rebrand: NTEF -> ECF

### PR DESCRIPTION
**Pull request by NopemanMcHalt**
:cl: Anonymous
wip: Changes NanoTrasen Expeditionary Fleet into Expeditionary Coalition Fleet. In nutshell, it's allows both Corporate Paramilitary and SolGov Military. ECF = (NTF + NTSC) + (SCGF + SCGEC).
tweak: Adds ECF into `SOLGOV_BRANCHES` loadout define, since it's mixture of Corporate and SolGov now.
/:cl:

thats_terror

### CODE REVIEW

**Comment by Difrit** [comment]
You, a corporate goon NTEF player complaining about people playing SolGov Fleet officers
farquaad

Me, a SolGov Fleet officer, changing the lore to suit my personal requirements
charming

**Comment by MarinaGryphon** [comment]
Do not merge without a database migration to change the name, otherwise it will cause issues with already-existing characters.

**Comment by MarinaGryphon** [comment]
Actually, unlike the genemod rename PR, I think this won't cause issues?

**Comment by MarinaGryphon** [comment]
Belay that. It will likely cause issues with loadout and skills. Please test it first.

File: maps/torch/loadout/_defines.dm [hunk 1]
```diff
@@ -54,7 +54,7 @@
 
 #define CIVILIAN_BRANCHES list(/datum/mil_branch/civilian, /datum/mil_branch/solgov)
 
-#define SOLGOV_BRANCHES list(/datum/mil_branch/solgov, /datum/mil_branch/marine_corps)
+#define SOLGOV_BRANCHES list(/datum/mil_branch/solgov, /datum/mil_branch/marine_corps, /datum/mil_branch/fleet)
```
Comment (MarinaGryphon) [inline maps/torch/loadout/_defines.dm]: This. This will break things for loadout and skills, to my knowledge.

**Comment by NopemanMcHalt** [comment]
0. The reason why it's called Salt PR is because it may cause salt (duh) or brew shitstorm if it will be merged, whenever its database issues or general idea.
1. https://github.com/BoHBranch/BoH-Bay/pull/363#pullrequestreview-357239846
2. Thanks for heads up about name change warning. I checked it and it's indeed messes up occupation data.
@ProbablyCarl

File: maps/torch/loadout/_defines.dm (hunk 1 above)
Comment (NopemanMcHalt) [reply maps/torch/loadout/_defines.dm]: It only checks if loadout item can be added on character if they're in X branch, and it really not should affect database ~~I hope~~.

**Comment by NopemanMcHalt** [comment]
Me, a contributor, being absent for a while and seeing a population boom and that nobody cares now so far:
image

**Comment by NopemanMcHalt** [comment]
It seems conversation about coalition is still on.

**Comment by Karl-Franz-I** [comment]
Commenting so I get email notifications about activity because I'd like to be there for discussions so we can fix this SolGov swept under the rug stuff

**Comment by N8-Toe** [comment]
I'm still 100% onboard with this

**Pull request closed without merging**